lib/api/web-element/commands/takeScreenshot.js

```diff
@@ -21,9 +21,9 @@
  * @memberof ScopedWebElement
  * @instance
  * @syntax browser.element(selector).takeScreenshot()
- * @see https://www.w3.org/TR/webdriver#dfn-take-element-screenshot
+ * @link /#dfn-take-element-screenshot
  * @returns {ScopedValue<string>}
  */
-module.exports.command = function() {
+module.exports.command = function () {
```

Comment:
I suggest, remove this unnecessary space between as it shows no significant change.